generate/genhcl/genhcl.go

```diff
@@ -82,9 +82,9 @@ const (
 	ErrDynamicConditionEval errors.Kind = "evaluating tm_dynamic.condition"
 )
 
-// Name of the HCL code.
-func (h HCL) Name() string {
-	return h.name
+// Label of the original generate_hcl block.
```

Comment:
Trying to abstract as a name only made it confusing to me, so I changed it to be more concrete.